onboarding_documentation/technical_documentation/metamist_docker/metamist_docker_tutorial.md

```diff
@@ -0,0 +1,763 @@
+# Introduction
+
+The aim of the following exercise is to allow you to get first hand experience working with the CPG codebase, the tools used to conduct analyses, as well as the supporting infrastructure that enables the management and utilisation of data.
+
```

Comment:
It would be nice to specify what 'first hand experience' will entail. What will the user be able to do at the end of the tutorial?  This would help the user select this tutorial if they were looking through documentation.  

How is the tutorial structured and what is the significance of that structure?